### ISSUE TITLE
emergency numbers can change dynamically

### ISSUE THREAD

**Issue by marcoscaceres** [sysapps/telephony#36]
Because the network can update the emergency numbers dynamically (as can changing physical location), should there be some kind of event for that?

**Comment by zolkis** [comment]
Yes, I also figured that we may need that event, although in most cases these number are coming from the modem/SIM, when the service gets enabled. Since we already have an event for that, I was on the lazy spec track, let's start without it and see if we can manage. But no big deal adding a dedicated event for it.
